src/main/frontend/src/utils/itemService.js

```diff
@@ -1,11 +1,50 @@
+import axios from "axios";
+import { handleResponse } from "./requestHandler";
 const host = "localhost:8080";
 
 export const itemsFetch = async (page, size) => {
-  const items = await fetch(`http://${host}/api/v1/items?page=${page}&size=${size}`)
+  const items = await fetch(
+    `http://${host}/api/v1/items?page=${page}&size=${size}`
+  );
   return items.json();
 };
 
 export const itemsFetchByDate = async (page, size, order, orderColumn) => {
```

Comment:
Haven't we agreed to merge fetchItems and fetchItemsByDate into one method?
Also rename it to `fetchItem` insteat of itemFetch